app/src/main/java/com/jjswigut/matters/ui/MatterListFragment.kt

```diff
@@ -75,16 +76,10 @@ class MatterListFragment : BaseFragment() {
     private fun handleAction(action: MatterAction) {
         when (action) {
             is MatterAction.MatterClicked -> {
-                val title = action.matter.matterTitle
-                val content = action.matter.matterContent
-                matter = Matter(content, title)
```

Comment:
so we already had the `matter` object we wanted defined in the action

creating a new matter here got rid of the ID of the matter we clicked on meaning we would never be able to update that matter anymore